packages/integrations/mdx/test/fixtures/mdx-frontmatter-injection/src/layouts/Base.astro

```diff
@@ -0,0 +1,17 @@
+---
+const defaults = { title: 'Frontmatter not passed to layout!' }
+const { frontmatter = defaults, content = defaults } = Astro.props;
+---
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>{frontmatter.title ?? content.title}</title>
```

Comment:
Shouldn't this only test `frontmatter.title`?

Reply:
`content` and `frontmatter` are the same thing! Other tests cover that these each exist. I can remove though since `content` is removed from docs